package.json

```diff
@@ -17,7 +17,9 @@
   },
   "homepage": "https://github.com/omriavidan/CallOfDuty#readme",
   "dependencies": {
+    "chai": "^4.2.0",
```

Comment:
Read about `devDependencies` in package.json
move chai and mocha there when you understand the devision between `dependencies` and `devDependencies`